### DISCUSSION TITLE
Welcome to ocpp Discussions!

### DISCUSSION THREAD

**Discussion by lbbrhzn** [lbbrhzn/ocpp#40] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by jeremycoppey** [comment]
Hi, 
Jérémy from Belgium. 
From the Dutch part of the country but I speak both (French and Dutch) languages fluently.
I have a [Schneider electric EVLink Wallbox](https://www.se.com/ww/en/product/EVH3S22P0CK/evlink-wallbox-plus---t2-attached-cable---3-phase---32a-22kw/)
I was looking for a way to connect my charger with my HASS system and this integration was the best I could find using the OCPP functionality  as there isn't another integration specific for this wallbox unlike other popular brands.

For the moment the wallbox is limited in the information it shares through OCPP so i hope future firmware updates fixes this.

I'm willing to help test and maintain documentation but I can't write python so I can't contribute in code.

Kind regards
Jérémy

**Reply by lbbrhzn** [reply]
Bienvenue au forum Jérémy!
It would be great of you could contirbute by testing and maintaining documentation...

**Comment by vandenbogerd** [comment]
Hi Jérémy,

I'm from Belgium too, and I also have the EVLink Wallbox.
Can you help me set it up? I'm trying for 2 days now, and it all seems to no avail...

TIA!

**Comment by PeterBorfai** [comment]
I'm looking for an EV charger that I can control with the OCPP integration in HA. Presently, I'm using a portable charger (max 2.2kW) which I switch on/off when I have solar power to spare. (The Huawei solar inverter integration is working great!) Can anyone recommend an EV charger that works correcty with the OCPP integration? Also, I'm trying to understand how maximum charge current is set. In a 3-phase system is the "maximum current" applied per phase or per charger? 
Any advice & recommendations would be welcome.
Peter
